### ISSUE TITLE
Tables | Project-Based Economy

### ISSUE THREAD

**Issue by arpithparikh** [JoinColony/colonyHackathon#44]
### Colony Hackathon Submission

**Project Title**

Tables | Project-Based Economy

---

**Project Description**

Tables will be an easy-to-use interface that allows any person to organize their project in a decentralized manner around any mission using the Colony framework.

**Project Repository**

**Team Members and Contact info**

kai@agnes.io

Please let me know if you're interested in contributing!

Outline for Tables:
- Each Table will represent a team working on a single project around a specific mission
- Distribution of reputation and payout will be facilitated through the Colony framework
- Ability for anyone without programming skills to build and join decentralized project-based teams around a specific mission
- Ability for users to both browse and be matched to particular teams based on their interests

**Comment by jrmoreau** [comment]
@arpithparikh thanks for your issue! Is your team accepting new members still?

**Comment by arpithparikh** [comment]
@jrmoreau Yes! Please reach out to us at kai@agnes.io for further details!
